cartography/intel/aws/ec2/vpc_peerings.py

```diff
@@ -0,0 +1,173 @@
+import logging
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_vpc_peerings(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_vpc_peering_connections()['VpcPeeringConnections']
+
+
+@timeit
+def load_vpc_peerings(neo4j_session, data, region, aws_account_id, aws_update_tag):
```

Comment:
Can you split this into one function per query to make it easier to test? Like one `load()` function that calls `load_{sub_object}()`?

Reply:
I hope I managed to fix this with the last commit #6dc5950 I was not fully sure what you meant by split